packages/eslint-plugin-nx/src/rules/nx-plugin-checks.ts

```diff
@@ -144,16 +146,16 @@ function normalizeOptions(
   return {
     ...base,
     executorsJson: base.executorsJson
-      ? `${sourceProject.data.root}/${base.executorsJson}`
+      ? resolve(`${sourceProject.data.root}/${base.executorsJson}`)
```

Comment:
This doesn't seem to do much?

Reply:
I'll double check if this is still needed - there was an issue when generating an Nx plugin at the root level that the lint rule was comparing `./generators.json` with `generators.json`